### PR TITLE
feat(MaStepper): Add space prop

### PR DESCRIPTION
**Pull request by sergigraciaj**
Add space prop in order to have more felxibility in how we separate the steps.

### CODE REVIEW

File: src/components/MaStepper/MaStepper.spec.js [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
 
 function renderComponent(props = {}) {
-  return render(MaStepper, {
+  const utils = render(MaStepper, {
     props,
     slots: { default: slot },
   })
+
+  const content = utils.getByText('Content 1').parentNode
+  const parentNode = utils.getByText('Content 1').parentNode
```
Comment (edufarre) [inline src/components/MaStepper/MaStepper.spec.js:56]: Aren't `content` and `parentNode` the same?

Comment (sergigraciaj) [reply src/components/MaStepper/MaStepper.spec.js]: True, thx!

**Review by icurdu — APPROVED** [review]
LGTM! I will add the maximum and minimum values in the documentation

**Pull request merged**